outplacement_invoice/models/account_invoice.py

```diff
@@ -1,28 +1,50 @@
 # -*- coding: utf-8 -*-
-
+"""
+Adds functions to handle invoices from Raindance server.
+"""
 import base64
 import logging
 import xmltodict
 
-from odoo import api, models, fields, tools, _
-
+from odoo import api, models, fields
+from odoo.addons.outplacement_invoice.static.svefaktura import SVEFAKTURA
 _logger = logging.getLogger(__name__)
 
 
 class AccountInvoice(models.Model):
+    """Handle Invoices from Raindance."""
     _inherit = "account.invoice"
     raindance_ref = fields.Char(string='Raindance ID')
 
     @api.model
     def cron_outplacement_invoice(self):
+        """Cron job entry point."""
+        self.outplacement_invoice(silent=True)
+
+    @api.model
+    def outplacement_invoice(self, silent=False):
+        """Get invoices from raindance and proccess them."""
         raindance = 'api.raindance.client.config'
+        if raindance not in self.env:
+            return
+        if not self.env[raindance].verify_config_is_set():
+            msg = 'Raindance Config is not set, cannot continue.'
+            _logger.warn(msg)
```

Comment:
Even if all the configuration for raindance is set, `self.env[raindance].verify_config_is_set` always returns `false`.
But the function works inside of `api.raindance.client.config` model:
( 
https://github.com/vertelab/odoo-api/blob/Dev-12.0-Fenix-Sprint-02/api_raindance_client/models/client_config.py
or on your machine `/usr/share/odoo-api/api_raindance_client/models/client_config.py`
)